cli/lsp/tsc.rs

```diff
@@ -5236,7 +5279,7 @@ mod tests {
   }
 
   #[tokio::test]
-  async fn test_completions() {
+  async fn test_completionss() {
```

Comment:
Accidental?

Reply:
oh good catch, didn't mean to commit that. 

It's kinda dumb, but sometimes when I want to run only a single test and the test happens to be a prefix of another test, i just add some chars to the end to make it unique. Forgot to revert it before staging the commit😅